multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -65,7 +71,7 @@ def __init__(
         teacher_model
             The teacher model in knowledge distillation.
         matches
-            Teacher/stduent layer matches to compute the intermediate loss.
```

Comment:
The `matches` was designed for the intermediate layers instead of the output features. We'd better not change the meaning here unless we actually use it in other ways.

Reply:
How about changing it back in the future?

Reply:
Will change it back in next pr